examples/ComplexExample/android/app/build.gradle

```diff
@@ -0,0 +1,140 @@
+apply plugin: "com.android.application"
```

Comment:
Instead of including the native ios/android files in the repo, you might want to consider putting them in the .gitignore file, and use `react-native eject` to create the native files as a build step. Not super feasible for real apps right now because the whole `app.json` thing is pretty minimal, but we can expand it as needed.